student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,6 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    //TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(), malloc_username_from_conf_file(), "MuktaDarekar");
```

Comment:
@MuktaDarekar you'd make the message here the mesage you want to print if the username match fails, so something like "Username doesn't match!"

Reply:
@MuktaDarekar you should free the string returned by `my_username` here.

Reply:
Modified both. Done.